### ISSUE TITLE
Need controls on rotating gallery

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1081]
The rotating gallery should have controls that allow the user to pause and advance the motion.

**Comment by jasoncorum** [comment]
@caseyedavis12 - Please take a look at the screenshots below and let me know if you'd like to change styling or placement. Thanks!

![screen shot 2017-02-14 at 10 56 42 am](https://cloud.githubusercontent.com/assets/3814190/22936804/962218c8-f2a4-11e6-985a-d93e123f69f1.png)

![screen shot 2017-02-14 at 10 56 57 am](https://cloud.githubusercontent.com/assets/3814190/22936811/98c5a482-f2a4-11e6-85e6-49054dad880c.png)

**Comment by caseyedavis12** [comment]
@jasoncorum I think it would be good to add the pause and play button just below the > button so that it doesn't take up extra horizontal space. What do you think? Can the button look more like the > button (white with no background)?

**Comment by jasoncorum** [comment]
@caseyedavis12 - Here's what that would look like. If this is good, I'll finish up coding. Thanks!
![screen shot 2017-02-17 at 8 21 26 am](https://cloud.githubusercontent.com/assets/3814190/23066773/3d45e318-f4ea-11e6-8c36-99b02c3f27cf.png)

**Comment by caseyedavis12** [comment]
@jasoncorum: the Pause button looks great. Can you send me a screenshot of the play button?

**Comment by jasoncorum** [comment]
Yup! Sorry I forgot that! 
![screen shot 2017-02-17 at 8 32 34 am](https://cloud.githubusercontent.com/assets/3814190/23067127/cf3f5226-f4eb-11e6-9bcb-e1af1b053645.png)

**Comment by caseyedavis12** [comment]
@jasoncorum: awesome!
